src/library_webgpu.js

```diff
@@ -2779,7 +2781,8 @@ var LibraryWebGPU = {
       "format": WebGPU.TextureFormat[
         {{{ gpu.makeGetU32('config', C_STRUCTS.WGPUSurfaceConfiguration.format) }}}],
       "usage": {{{ gpu.makeGetU32('config', C_STRUCTS.WGPUSurfaceConfiguration.usage) }}},
-      "alphaMode": "opaque",
+      "alphaMode": WebGPU.AlphaMode[
```

Comment:
We tend to prefer single quotes for strings in the JS library.

Reply:
This would make it inconsistent with the rest of the file, so I'd like to leave it for now if that's OK. I've filed #22496 since we now have a small backlog of assorted fixes we should do.

Reply:
sgtm